main/process-tx.zkasm

```diff
@@ -144,6 +144,8 @@ endCalldataIntrinsicGas:
         $ => B                          :MLOAD(txGasPrice)
         A                               :MSTORE(arithA)
         B                               :MSTORE(arithB), CALL(mulARITH)
+        $ => D                          :MLOAD(arithOverflow)
+        -D                              :JMPN(invalidIntrinsicTxGasOverflow)
```

Comment:
Whit the new static address of jumps you can make the line like this (on step less):
$ => D                          :MLOAD(arithOverflow), JMPN(invalidIntrinsicTxGasOverflow)